opendg/graph.py

```diff
@@ -45,6 +47,39 @@ def from_csv(
         events = read_csv(file_path, *args, **kwargs)
         return cls(events, time_delta)
 
+    @classmethod
+    def from_pandas(
+        cls,
+        df: pd.DataFrame,
+        src_col: str,
+        dst_col: str,
+        time_col: Optional[str] = None,
+        edge_feature_col: Optional[str] = None,
+        time_delta: Optional[TimeDeltaDG] = None,
+    ) -> 'DGraph':
+        r"""Load a Dynamic Graph from a Pandas DataFrame.
+
+        Args:
+            df (pd.DataFrame): The dataframe to read from.
+            src_col (str): Column in the dataframe corresponding to the source edge id.
+            dst_col (str): Column in the dataframe corresponding to the destination edge id.
+            time_col (str): Column in the dataframe corresponding to the timestamp.
+            edge_feature_col (Optional[str]): Optional column in the dataframe corresponding to the edge features.
+            time_delta (Optional[TimeDeltaDG]): Describes the time granularity associated with the event stream.
+                If None, then the events are assumed 'ordered', with no specific time unit.
+
+        Returns:
+           DGraph: The newly constructed dynamic graph.
+        """
+        non_ordered_time_delta = time_delta is not None and not time_delta.is_ordered
```

Comment:
```suggestion
        ordered_time_delta = time_delta is None or time_delta.is_ordered
```
much simpler to read